### ISSUE TITLE
Push service documentation

### ISSUE THREAD

**Issue by Fuseteam** [ubports/docs.ubports.com#611]
it appears there is some documentation about our push service over at https://gitlab.com/ubports/development/core/lomiri-push-service/-/tree/main/docs

I think we should incorporate it, but i'm not sure how to approach it yet

**Comment by Fuseteam** [comment]
wouldn't be better to have it be readable in the docs? i think it'll read easier

**Comment by doniks** [comment]
Another thought, *if* lps devs rather have it in their repo, but it's hard to read, then it might be best to improve the language at the source ... I mean it's all *us* ;)

**Comment by Fuseteam** [comment]
yeah that is fair granted the docs hasn't changed in years, i believe the maintainer is @Flohack74

**Comment by Flohack74** [comment]
Did not even know that docs are esisting :D
